scripts/parseXML.js

```diff
@@ -19,7 +73,7 @@ function getImages(products) {
  * 
  * @param {string} path path of the XML file
  */
-module.exports = (path) => {
+module.exports = (path, objectType) => {
```

Comment:
what object type please complete JSDOC

Reply:
I'd prefer to create 2 functions exports rather than passing this extra parameter

Reply:
Separate functions have been created and exported.